spec/models/concerns/energy_tariff_holder_spec.rb

```diff
@@ -60,6 +62,36 @@
         expect(tariff_holder.energy_tariff_meter_attributes.first).to be_a MeterAttribute
       end
 
+      it 'maps the tariffs to meter attribute filtering by meter type and applies to' do
+        expect(tariff_holder.energy_tariffs.count).to eq(4)
+
+        expect(tariff_holder.energy_tariffs.both.count).to eq(2)
+
+        expect(tariff_holder.energy_tariffs.electricity.both.count).to eq(1)
+        expect(tariff_holder.energy_tariffs.electricity.half_hourly.count).to eq(1)
+        expect(tariff_holder.energy_tariffs.electricity.non_half_hourly.count).to eq(1)
+
+        expect(tariff_holder.energy_tariffs.gas.both.count).to eq(1)
+        expect(tariff_holder.energy_tariffs.gas.half_hourly.count).to eq(0)
+        expect(tariff_holder.energy_tariffs.gas.non_half_hourly.count).to eq(0)
```

Comment:
These aren't actually testing anything, they're just asserting that the dummy data exists? So am not sure they're adding anything.